src/vivarium_gates_nutrition_optimization_child/components/lbwsg.py

```diff
@@ -162,11 +171,28 @@ def get_birth_exposure(self, axis: str, index: pd.Index) -> pd.DataFrame:
 
         assigned_categories = list(lbwsg_categories) * (2 * num_repeats)
         pop["lbwsg_category"] = assigned_categories
+        self.population_view.update(pop[["age_bin", "lbwsg_category"]])
 
+        birth_exposures = {
+            self.exposure_column_name(axis): self.birth_exposures[
+                self.birth_exposure_pipeline_name(axis)
+            ](pop_data.index)
+            for axis in self.AXES
+        }
+        self.population_view.update(pd.DataFrame(birth_exposures))
```

Comment:
Is there a reason why you're updating twice instead of just once at the end?

Reply:
Oh - they're needed in the pipeline. I'd add a comment saying that since it deviates from our usual pattern of updating once at the end